lsp-erlang.el

```diff
@@ -38,12 +38,25 @@
   :risky t
   :type 'file)
 
+(defcustom lsp-erlang-server-connection-type
+  'tcp
+  "Type of connection to use with the Erlang Language Server: tcp or stdio"
+  :group 'lsp-erlang
+  :risky t
+  :type 'string)
+
 (defun lsp-erlang-server-start-fun (port)
   `(,lsp-erlang-server-path
-    ,(number-to-string port)))
+    "--transport" "tcp"
+    "--port" ,(number-to-string port)))
+
+(defun lsp-erlang-server-connection ()
```

Comment:
Here you could return a function that will return the function to create the connection based on the connection-type.

Reply:
(not necessary to address).